src/core/packages/ui-settings/server-internal/src/saved_objects/migrations.ts

```diff
@@ -10,8 +10,26 @@
 import type {
   SavedObjectUnsanitizedDoc,
   SavedObjectSanitizedDoc,
+  SavedObjectsModelVersionMap,
 } from '@kbn/core-saved-objects-server';
 
+/**
+ * @note Since all `uiSettings` migrations are added to the same migration function,
+ * while not required, grouping settings by team, using a consistent naming prefix,
+ * is good practice. For example: `ml:<setting-name>`.
+ */
+
+export const modelVersions: SavedObjectsModelVersionMap = {
+  /** First model version picksup from the last legacy migration in 8.9.0 */
+  1: {
+    changes: [],
+  },
+};
```

Comment:
I know it's supposed to be "common knowledge", but maybe we should add a comment to remind that any new addition should introduce a new version?